trunk/Public/DynamicConditionsPublic.php

```diff
@@ -248,10 +248,12 @@ private function convertAcfDate( $id, array $data ) {
      * @param Locations_Manager $locationManager
      */
     public function checkPopupsCondition( $locationManager ) {
-        if ( ! empty( Plugin::$instance->preview ) && Plugin::$instance->preview->is_preview_mode() ) {
+        if ( !empty( Plugin::$instance->preview ) && Plugin::$instance->preview->is_preview_mode() ) {
+            return;
+        }
+        if ( !empty( Plugin::$instance->editor ) && Plugin::$instance->editor->is_edit_mode() ) {
```

Comment:
Two `if` statements right after another that do the same thing? I'd prefer combining the conditions.

Reply:
I think with two conditions, it is more readable

Reply:
You could always put the conditions in separate methods?

Reply:
Done